backends/advanced/src/scripts/cleanup_state.py

```
@@ -421,6 +431,47 @@ async def _export_memories(self, qdrant_client: AsyncQdrantClient, stats: Cleanu
         except Exception as e:
             logger.warning(f"Failed to export memories: {e}")
 
+    def _export_neo4j(self, stats: CleanupStats):
```

Ruff (0.14.14)

[warning] Unused method argument: `stats`

(ARG002)